ast/functions_test.go

```diff
@@ -76,6 +76,9 @@ func (ts *testFunctionsSuite) TestFuncCallExprRestore(c *C) {
 		{"nextval(seq)", "NEXTVAL(`seq`)"},
 		{"lastval(seq)", "LASTVAL(`seq`)"},
 		{"setval(seq, 100)", "SETVAL(`seq`, 100)"},
+		{"next value for seq", "NEXTVAL(`seq`)"},
+		{"next value for sequence", "NEXTVAL(`sequence`)"},
+		{"NeXt vAluE for seQuEncE2", "NEXTVAL(`seQuEncE2`)"},
```

Comment:
Please add test cases for the `` `db`.`tbl` `` form.

Reply:
Make sense, addressed.